Documentation.md

```diff
@@ -835,3 +835,65 @@ call `.withObject(o)` to add an arbitrary object (`o`) to the logs.
 
 Finally, you can trigger the logs to be downloaded to the agent's machine in JSON form by calling `connect.getLog().download()`.
 
+## CCP Connectivity Errors and Events
+The following errors are related to agent connectivity in the CCP.
+
+### `unsupported_browser` 
+Agent is using an unsupported browser. Only the latest 3 versions of Chrome or Firefox is supported. Upgarde the agent's browser to resolve this error. See [Supported browsers](https://docs.aws.amazon.com/connect/latest/adminguide/what-is-amazon-connect.html#browsers)  for more information.
+
+### `microphone_not_shared`
+The microphone does not have permission for the site on which the CCP is running. For Google Chrome steps, see [Use your camera and microphone in Chrome](https://support.google.com/chrome/answer/2693767?hl=en). For Mozilla Firefox steps, see [Firefox Page Info window](https://support.mozilla.org/en-US/kb/firefox-page-info-window).
+
+### `signalling_handshake_failure`
+Error connecting the CCP to the telephony system. To resolve, try the action again, or wait a minute and then retry. If the issue persists, contact support.
+
+### `signalling_connection_failure`
+Internal connection error occurred. To resolve, try the action again, or wait a minute and then retry. If the issue persists, contact support.
+
+### `ice_collection_timeout`
+Error connecting to the media service. To resolve, try the action again, or wait a minute and then retry. If the issue persists, contact support.
+
+### `user_busy_error`
+Agent has the CCP running in 2 distinct browsers at the same time, such as Chrome and Firefox. Use only one browser at a time to log in to the CCP.
+
+### `webrtc_error`
+An issue occurred due to either using an unsupported browser, or a required port/protocol is now open, such as not allowing UDP on port 443. To resolve, confirm that the agent is using a supported browser,a nd that all traffic is allowed for all required ports and protocols. See [CCP Networking](https://docs.aws.amazon.com/connect/latest/adminguide/troubleshooting.html#ccp-networking) and [Phone Settings](https://docs.aws.amazon.com/connect/latest/userguide/agentconsole-guide.html#phone-settings) for more information. 
```

Comment:
This should be "is not open" instead of "is now open"?

Reply:
Also a typo at "browser,a nd"

Reply:
thank you!